include/umf.h

```diff
@@ -15,4 +15,13 @@
 #include <umf/mempolicy.h>
 #include <umf/memspace.h>
 
+// Increment the usage reference counter and initialize the global state of libumf
+// if the usage reference counter was equal 0.
```

Comment:
was equal 0 -> is equal to 0

Reply:
Done

Reply:
This one wasn't updated, please fix also similar comments in libumf.c

Reply:
There should be `was`, not `is`.

Reply:
Done